.pre-commit-config.yaml

```diff
@@ -18,5 +18,5 @@ repos:
   - id: goimports
   - id: gofmt
   - id: golangci-lint
-  - id: gotests
+#  - id: gotests
```

Comment:
gotests target only run "go test", it doesn't start docker-compose.
We need to add a custom pre commit hook on webhooks to start them.
Also, i think we should avoid using docker for testing as much as possible.

Reply:
Yes you're right!